src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -23,6 +23,41 @@ namespace trace {
 
 CorProfiler* profiler = nullptr;
 
+ WSTRING skip_assemblies[] {
```

Comment:
Thanks for moving this! Initializing this array on every `ModuleLoadFinished()` was silly. With this here, can we remove the one in line 373?

Reply:
arrg! I had removed the other one (line 373) earlier when I had a sub-branch. Thanks for catching that!